### ISSUE TITLE
fileTypes property not being adhered to

### ISSUE THREAD

**Issue by keyboSlice** [23/resumable.js#250]
Hi there, this might be an error on my part - however i've specified an array of file types (in the form of mime types) that I want to permit users to upload, my setup looks like this:

    this.r = new Resumable (
		{
			target: "/video/upload",
			query: { _token: $('meta[name=token]').attr ("content"), file_type: self.fileType },
			simultaneousUploads: 1,
			maxFiles: 1,
			chunkSize: 1 * 1024 * 1024,
			permanentErrors: [403, 404, 415, 500, 501],
			fileTypes: this.fileTypes,
			fileTypeErrorCallback: function (file, errorCount)
			{
				bootbox.alert ("Sorry that file format is not supported.<br>Valid file extensions are: .mov, .mp4, .mpeg4, .avi, .wmv, .mpegps, .flv, .3gpp, .webm");
			}
		});

and this.fileTypes =  ["video/webm", "video/ogg", "video/3gpp", "video/mpeg4-generic", "video/mp4", "video/quicktime", "video/3gpp2"];

however it's allowing me to add image/jpeg etc, can anyone please advise as to what i'm doing wrong?

**Comment by steffentchr** [comment]
@keyboSlice Unfortunately there's no really good way to do this with mimetypes as you suggest: 
* Either you rely on [the browser's built-in support for file type accept](http://www.w3schools.com/tags/att_input_accept.asp), which doesn't consistently allow JavaScript callbacks and a good UI for hidden `input` elements.
* Or you maintain your own list.

None of these are particularly helpful: The first one would have Resumable be unusable when unaccepted files are dropped in. The second one would force us and ultimately the end-developers to keep a map of extension-to-mime-type up to date. So instead of doing that, we just let you specify the extensions instead.

In short: `fileTypes` should be something like `['mp4', 'wmv', ...]` instead of mimes.

**Comment by keyboSlice** [comment]
extensions works for me, thanks for your help :)
